### PR TITLE
Almanac nouns

### PR DESCRIPTION
**Pull request by Hiinky**

### CODE REVIEW

File: profiles/base.yaml [hunk 1]
```diff
@@ -1221,6 +1221,8 @@ forge_container:
 spell_cycle:
 
 restock_shop:
+#Almanac noun
+almanac: almanac
```
Comment (Sarvatt) [inline profiles/base.yaml]: almanac_noun: would be more fitting here

**Comment by Sarvatt** [comment]
the base-hunting changes need to go

File: profiles/base.yaml (hunk 1 above)
Comment (Hiinky) [reply profiles/base.yaml]: I can update. just followed the textbook:

**Comment by Sarvatt** [comment]
i guess it doesnt matter, you are literally the only person who will need this. i was going off the climbing rope nound change someone else needed for their unique one before

File: profiles/base.yaml (hunk 1 above)
Comment (rpherbig) [reply profiles/base.yaml]: Agreed 👍

**Comment by rpherbig** [comment]
Please merge master into your branch to get rid of the `base-hunting` changes - or were those part of this PR?

**Pull request closed without merging**